lib/wallet/src/Cardano/Wallet/Shelley/Transaction.hs

```diff
@@ -713,9 +713,11 @@ mkUnsignedTransaction networkId stakeCred ctx selection = do
         Just action -> do
             let certs = case stakeCred of
                     Left xpub ->
-                        certificateFromDelegationAction era (Left xpub) action
+                        certificateFromDelegationAction
+                            @era (Left xpub) action
```

Comment:
```suggestion
                        certificateFromDelegationAction (Left xpub) action
```

as above

Reply:
This particular type application was redundant. I removed it in https://github.com/cardano-foundation/cardano-wallet/compare/642df6baeeaa123bfca4dd48c14e81b58c2278a1..55ac25413a88da79a89b5848950a3a575fd6fc8c.